src/main/resources/application.yml

```diff
@@ -75,7 +74,6 @@ registry:
   agencyTokensByDomainAndOrganisationFormat: "${registry.serviceUrl}/agencyTokens?domain=%s&code=%s"
   organisationalUnitsFlatUrl: "${registry.serviceUrl}/organisationalUnits/flat"
   updateSpacesAvailableUrl: "${registry.serviceUrl}/agencyTokens"
-  getOrganisationUrl: "${registry.serviceUrl}/civilServants/org/%s"
   updateOrganisationUrl: "${registry.serviceUrl}/civilServants/org"
```

Comment:
Do we need this update endpoint url? Can the endpoint be removed from csrs as well?